docs/faststore/docs/go-live/1-configuring-external-dns.mdx

```diff
@@ -45,19 +45,35 @@ Now, you must set up your VTEX account to use the DNS records created in the pre
 4. In the **Hosts** field, enter your main domain (example: `www.mystore.com`) into the list.
 5. Click `Add`.
 6. Add the `secure` subdomain (example: `secure.mystore.com`) to the list.
+   
    ![host-configuration](https://vtexhelp.vtexassets.com/assets/docs/src/account-settings-new___2e13bee3041e8505922a737b94652ca6.png)
 
 7. Add the root domain as well; for example, if your domain is `www.mystore.com`, add `mystore.com`. For a complete setup, you must add three hosts for each domain:
-   - `www.mystore.com`  (store domain)
+   - `www.mystore.com` (store domain)
    - `mystore.com` (root domain)
    - `secure.mystore.com` (checkout/myaccount subdomain)
 
-> ⚠️ To ensure a seamless checkout experience, all subdomains used by your store must be added to the host list. For more information, see the [Adding subdomains to the host list](#adding-subdomains-to-the-host-list) section.
+<details>
+<summary><b>Stores hosted on Vercel</b></summary>
+
+Stores hosted on [Vercel](https://vercel.com/) can benefit from a single domain. This feature simplifies domain management by routing Checkout and My Account pages through the store’s primary domain (e.g., `www.domain.com`), removing the need for a separate `secure.domain.com` subdomain.
+
+Add the hosts for the domain as shown in the example below:
```

Comment:
Since we are using this approach of a single domain for stores hosted on Vercel, are these instructions correct when setting up the hosts on VTEX Admin?

![single-domain-step3](https://github.com/user-attachments/assets/1d27b33a-a4f1-42b4-967f-60522f9e9479)